lua/lspconfig/server_configurations/cssmodules_ls.lua

```diff
@@ -0,0 +1,32 @@
+local util = require 'lspconfig.util'
+
+local bin_name = 'cssmodules-language-server'
+local cmd = { bin_name }
+
+if vim.fn.has 'win32' == 1 then
+  cmd = { 'cmd.exe', '/C', bin_name }
+end
+
+return {
+  default_config = {
+    cmd = cmd,
+    filetypes = { 'javascript', 'javascriptreact', 'typescript', 'typescriptreact' },
+    root_dir = util.find_package_json_ancestor,
+  },
+  docs = {
+    package_json = 'https://github.com/antonk52/coc-cssmodules/blob/master/package.json',
```

Comment:
I don't think we should use a coc extension's package.json

Reply:
Yeah now when I look closer it seems like that package.json is providing coc-style settings. It'd probably be better to file an issue/PR in the [standalone language server](https://github.com/antonk52/cssmodules-language-server) repo and try to introduce the LSP metadata in that package.json as well. I'll remove this `package_json` for now

Reply:
Hmm I took a look at a few other package_json entries for reference. It seems like they suffers the same issue, but s/coc/VSCode/ (for example intelephense & solargraph)?  Or maybe I misunderstand the `package_json.contributes` schema

Reply:
It depends, sometimes these mix vscode and legitimate server settings, in which case I haven't particularly cared.

Reply:
It's actually just good ole JSONSchema https://json-schema.org/

Reply:
The unfortunate thing is that many LSP server devs have ended up merging the LSP server settings with the VSCode-extension settings, which is what ends up being documented in a JSONSchema

Reply:
pylsp is one of few servers that provide a separate schema for the server itself, maybe it could be used as reference - https://github.com/python-lsp/python-lsp-server/blob/develop/pylsp/config/schema.json

Reply:
Thank you for the info. I am going to look into it and updating the package.json accordingly👍